shared/pubsub.js

```diff
@@ -243,7 +244,7 @@ const defaultClientEventHandlers = {
   error (event: Event) {
     const client = this
 
-    console.log('[pubsub] Event: error', event)
+    console.error('[pubsub] Event: error', event)
```

Comment:
Not all `error` events should be logged with `console.error`, for example every failed connection attemp generates one such event

Reply:
noted! will be changed to `console.warn`